[2024-05-04T07:29:26.265+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-04T07:29:26.279+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.is_weather_api_ready manual__2024-05-04T07:29:22.919450+00:00 [queued]>
[2024-05-04T07:29:26.288+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.is_weather_api_ready manual__2024-05-04T07:29:22.919450+00:00 [queued]>
[2024-05-04T07:29:26.288+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 3
[2024-05-04T07:29:26.312+0000] {taskinstance.py:2327} INFO - Executing <Task(HttpSensor): is_weather_api_ready> on 2024-05-04 07:29:22.919450+00:00
[2024-05-04T07:29:26.334+0000] {standard_task_runner.py:63} INFO - Started process 26670 to run task
[2024-05-04T07:29:26.341+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'weather_dag', 'is_weather_api_ready', 'manual__2024-05-04T07:29:22.919450+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpcv3cn0tx']
[2024-05-04T07:29:26.342+0000] {standard_task_runner.py:91} INFO - Job 37: Subtask is_weather_api_ready
[2024-05-04T07:29:26.407+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_dag.is_weather_api_ready manual__2024-05-04T07:29:22.919450+00:00 [running]> on host ip-172-31-26-226.eu-north-1.compute.internal
[2024-05-04T07:29:26.506+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='myemail@domain.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='is_weather_api_ready' AIRFLOW_CTX_EXECUTION_DATE='2024-05-04T07:29:22.919450+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-04T07:29:22.919450+00:00'
[2024-05-04T07:29:26.509+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-04T07:29:26.509+0000] {baseoperator.py:399} WARNING - HttpSensor.execute cannot be called outside TaskInstance!
[2024-05-04T07:29:26.509+0000] {http.py:137} INFO - Poking: /data/2.5/weather?q=Portland&APPID=7da7887d8eb0a81af4955a3474856f9a
[2024-05-04T07:29:26.518+0000] {base.py:84} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-05-04T07:29:26.743+0000] {base.py:294} INFO - Success criteria met. Exiting.
[2024-05-04T07:29:26.745+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-04T07:29:26.750+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=is_weather_api_ready, execution_date=20240504T072922, start_date=20240504T072926, end_date=20240504T072926
[2024-05-04T07:29:26.795+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-04T07:29:26.811+0000] {logging_mixin.py:188} WARNING - /home/ubuntu/airflow_environment/lib/python3.12/site-packages/airflow/models/baseoperator.py:1296 AirflowProviderDeprecationWarning: Call to deprecated class SimpleHttpOperator. (Class `SimpleHttpOperator` is deprecated and will be removed in a future release. Please use `HttpOperator` instead.)
[2024-05-04T07:29:26.836+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-04T07:29:26.841+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
